src/Dropdown/Dropdown.js

```diff
@@ -63,6 +100,28 @@ export default class Dropdown extends Util.mixin(BindMixin) {
     }, this);
   }
 
+  getSelectedHtml(id, items) {
+    let obj = Util.find(items, function (item) {
+      return item.id === id;
+    });
+
+    return obj.selectedHtml || obj.html;
+  }
+
+  getSpaceAroundDropdown() {
+    let dropdownWrapper = React.findDOMNode(this.refs.dropdownWrapper);
+    let position = dropdownWrapper.getBoundingClientRect();
+    let viewportHeight = global.window.innerHeight;
+
+    // Calculate the distance from the top of the viewport to the top of the
+    // dropdown button as well as the distance from the bottom of the viewport
+    // to the bottom of the dropdown button.
+    return {
+      top: position.top,
```

Comment:
Nope! Good call, I forgot that we're just testing the distance from the bottom of the viewport.